packages/xrpl/test/client/requestNextPage.test.ts

```diff
@@ -57,4 +57,31 @@ describe('client.requestNextPage', function () {
       'response does not have a next page',
     )
   })
+
+  // TODO: Write this test to verify multiple types of commands can be run
```

Comment:
Can you create a follow up task to do this?

Reply:
https://github.com/XRPLF/xrpl.js/issues/2384